connect/common/src/test/resources/query-tests/explain-results/function_base64.explain

```diff
@@ -1,2 +1,2 @@
-Project [base64(cast(g#0 as binary)) AS base64(CAST(g AS BINARY))#0]
+Project [static_invoke(Base64.encode(cast(g#0 as binary), false)) AS base64(CAST(g AS BINARY))#0]
```

Comment:
we can omit `false` here

Reply:
How do I omit it? `false` also seems to be preserved in `function_encode.explain`.

https://github.com/apache/spark/blob/4c7edd2a20480a8521fcc88a966b22619143aebd/connect/common/src/test/resources/query-tests/explain-results/function_encode.explain#L1

Reply:
override toString

Reply:
I have found some other functions also having such issue, we can fix them together later